rpc/documentation/api.md

```diff
@@ -1476,3 +1477,86 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+## `TicketBuyerLoaderService`
+
+The `TicketBuyerLoaderService` service provides RPC clients with the ability to
+launch the automatic ticket purchaser with given initial configuration
+parameters, and to stop it.  It has no dependencies and is always running.
+
+**Methods:**
+
+- [`StartTicketPurchase`](#startticketpurchase)
+- [`StopTicketPurchase`](#stopticketpurchase)
+
+### Methods
+
+#### `StartTicketPurchase`
+
+The `StartTicketPurchase` method starts an automatic ticket purchaser with the given parameters.
+
+**Request:** `StartTicketPurchaseRequest`
+
+- `bytes passphrase`: The private passphrase to unlock the wallet.
+
+- `uint32 account`: The account number to use for purchasing tickets.
+
+- `int64 balance_to_maintain`: The minimum amount of funds to never dip below when purchasing tickets.
+
+- `int64 max_fee_per_kb`: The maximum ticket fee amount per KB.
+
+- `double max_price_relative`: The scaling factor for setting the maximum ticket price, multiplied by the average price.
+
+- `int64 max_price_absolute`: The maximum absolute ticket price.
+
+- `string voting_address`: The address to send all ticket outputs.
+
+- `string pool_address`: The ticket pool address where ticket fees will go to.
```

Comment:
still wrong.. the ticket pool doesn't have addresses.

This is an option for stakepools.